File: UserAccount.py
class UserAccount:
    def __init__(self, username, password):
        self.username = username
        self.password = password

    def setUsername(self, username):
        self.username = username

    def setPassword(self, password):
        self.password = password

    def getUsername(self):
        return self.username

    def getPassword(self):
        return self.password

File: UserAccount.cpp
#include "UserAccount.h"
#include <vector>
#include <string>
#include <iostream>
#include <fstream>

using namespace std;

UserAccount::UserAccount()
{
    username = "";
    password = "";
}

void UserAccount::setUsername(string user)
{
    this->username = user;
}

void UserAccount::setPassword(string pass)
{
    this->password = pass;
}

string UserAccount::getUsername()
{
    return username;
}

string UserAccount::getPassword()
{
    return password;
}
